UIAutomationTests/TestsUsernamePassword.cs

```diff
@@ -0,0 +1,331 @@
+﻿using System;
+using System.IO;
+using Xunit;
+
+namespace UIAutomationTests
+{
+    //https://xunit.net/docs/running-tests-in-parallel
+    //We don't want to run them in parallel (since we're doing automation UI testing) so we add the tests to the same test collection
+    [Collection("NotParallel")]
+    public class TestsUsernamePassword : IClassFixture<TestFixture>
```

Comment:
would be nice to separate the tests case that are not authentication-specific to another class

Reply:
I agree. I'll split them up.